README.md

```diff
@@ -6,65 +6,64 @@
+|                          | [secio](https://github.com/libp2p/specs/blob/master/secio/README.md) **(deprecated)**           |  :green_apple:   |
+| **Protocol Select**      | [multistream](https://github.com/multiformats/multistream-select)                               |  :green_apple:   |
+| **Stream Multiplexing**  | [yamux](https://github.com/libp2p/specs/blob/master/yamux/README.md)                            |     :lemon:      |
+|                          | [mplex](https://github.com/libp2p/specs/blob/master/mplex/README.md)                            |  :green_apple:   |
+| **NAT Traversal**        | [circuit-relay-v2](https://github.com/libp2p/specs/blob/master/relay/circuit-v2.md)             |                  |
+|                          | [autonat](https://github.com/libp2p/specs/tree/master/autonat)                                  |                  |
+|                          | [hole-punching](https://github.com/libp2p/specs/blob/master/connections/hole-punching.md)       |                  |
+| **Discovery**            | [bootstrap](https://github.com/libp2p/specs/blob/master/kad-dht/README.md#bootstrap-process)    |                  |
+|                          | random-walk                                                                                     |                  |
+|                          | [mdns-discovery](https://github.com/libp2p/specs/blob/master/discovery/mdns.md)                 |     :lemon:      |
+|                          | [rendezvous](https://github.com/libp2p/specs/blob/master/rendezvous/README.md)                  |                  |
+| **Peer Routing**         | [kad-dht](https://github.com/libp2p/specs/blob/master/kad-dht/README.md)                        |                  |
+| **Publish/Subscribe**    | floodsub                                                                                        |     :lemon:      |
+|                          | [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub)                       |  :green_apple:   |
+| **Storage**              | record                                                                                          |                  |
+| **Other protocols**      | [ping](https://github.com/libp2p/specs/blob/master/ping/ping.md)                                |  :green_apple:   |
+|                          | [identify](https://github.com/libp2p/specs/blob/master/identify/README.md)                      |  :green_apple:   |
+
+Legend:
+- :green_apple: - tested in production
+- :lemon: - prototype or beta, not tested in production
+- :tomato: - in progress 
 
 ## Gossip simulator
 
 Deterministic Gossip simulator which may simulate networks as large as 10000 of peers
 
 Please check the Simulator [README](tools/simulator/README.md) for more details
 
+## Android support
+
+The library is basically being developed with Android compatibility in mind. However we don't aware of any library usage
```

Comment:
`However we are not aware of anyone using it in production` sounds better in my opinion